kvmagent/kvmagent/plugins/deip.py

```
@@ -334,6 +334,12 @@ def create_ebtable_rule_if_needed(table, chain, rule, at_head=False):
                 else:
                     bash_errorout(EBTABLES_CMD + ' -t {{table}} -A {{chain}} {{rule}}')
 
+        @bash.in_bash
+        def ensure_ebtable_rule_at_head(table, chain, rule):
```

Ruff (0.15.2)

[warning] Unused function argument: `table`

(ARG001)